slackbot_using_azure_openai/README.md

```diff
@@ -0,0 +1,38 @@
+# Chatbot service for Slack using Azure OpenAI
+
+## Use Case
+The service responds to questions asked in Slack chanels using the slash command (`/<COMMAND> question`), using the Azure OpenAI chat model by maintainging a history of 25 messages.
+
+## Prerequisites
+* Azure OpenAI service
+* Slack account and app
+
+## Setting up Azure OpenAI service
+1. Follow the instructions given in [Azure Documentation](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/create-resource?pivots=web-portal) to create an Azure OpenAI resource and deploy a text completion model.
+2. In the Azure Portal, go to **Keys and Endpoint** in the **Resource Management** section of the resource and obtain the service URL (**Endpoint**) and API key (either **KEY1** or **KEY2**).
+3. Go to **Deployments** in the **Resource Management** section and obtain the name of the text completion model which was deployed in step 1.
+
+## Setting up Slack account and app
+1. Sign up or sign in to [Slack](https://slack.com/get-started#/createnew).
```

Comment:
Shall we change this to `Sign up and log into`?

Reply:
Addressed in [8b414ef](https://github.com/ballerina-guides/ai-samples/pull/64/commits/8b414efd4c7f6c79f5e6913b45a946ab677e04f3)